README.md

```diff
@@ -14,74 +17,78 @@
 Backed by Y Combinator
 </div>
 
----
+# Fern
+
+Fern is an open source toolkit for designing, building and consuming REST APIs. Fern can be used in your existing projects through OpenAPI, and integrate effortlessly into your existing workflows. It can also serve as the basis of your API development, by describing your endpoints in a Fern specification, and having Fern generate your API backend code, as well as SDKs.
+
```

Comment:
I think we should get to the punchline a little earlier: 

Fern is an open source toolkit for designing, building and consuming REST APIs. With Fern, you can generate documentation, client libraries and boilerplate for your backend server in your chosen framework (e.g. FastAPI, Express). 

Fern can be used in your existing projects through OpenAPI or if you choose, you can describe your API in the Fern specification. In this sense, Fern is an alternative to the OpenAPI standard without seeking to replace it.

This repository contains the Fern compiler. The compiler takes...

Reply:
@aevitas what do you think about this for the intro?